source/release-notes/1.8.txt

```diff
@@ -21,9 +21,91 @@ Release Notes for mongosync 1.8
 
 **Upcoming**
 
-New Features:
+Embedded Verifier
+~~~~~~~~~~~~~~~~~
+
+Starting in 1.8, ``mongosync`` includes an optional embedded
+verifier that performs a series of checks against the source and 
+destination clusters to verify a successful sync.
+
+For more information, see :ref:`c2c-embedded-verifier`.
+
+Other Notes
+~~~~~~~~~~~
+
+Updates to ``mongosync`` ``/progress`` API endpoint:
+
+- The ``/progress`` endpoint now returns a top-level boolean 
+  ``success`` field.
+
+- The ``/progress`` endpoint now returns the integer 
+  ``totalEventsApplied`` field, which approximates the number of 
+  change events applied to a destination cluster by an instance of 
+  ``mongosync``.
+
+- The ``/progress`` endpoint now reports per-partition progress 
+  tracking with the following changes:
+
+  - Calling ``/progress`` before ``mongosync`` reaches the collection 
+    copy phase will return 0 for ``estimatedCopiedBytes`` and 1 for 
+    ``estimatedTotalBytes``.
+
+  - ``estimatedTotalBytes`` may change throughout the collection copy 
+    phase if documents are inserted or deleted on the source cluster.
+
+  - ``estimatedCopiedBytes`` will never be greater than 
+    ``estimatedTotalBytes``. Progress will reach 100% at the end of 
+    the collection copy phase 
+    (``estimatedCopiedBytes`` = ``estimatedTotalBytes``).
+
+  - When performing a live upgrade from an earlier version to 1.8.0 or 
+    higher, the collection copy data will start over from 
+    ``[0 bytes copied / 1 bytes total]``. After a live upgrade, 
+    ``/progress`` only reports the progress of data copied after the 
+    upgrade completed.
+
+Optimizations:
+
+- Significantly reduced the time needed to create partitions and added logging 
```

Comment:
(tense)
Present tense here. Maybe consider rewording a smidge. Something like:
Fixes and enhancements to reduce time needed ...

Reply:
I adopted your wording, but retained `significantly` since this change results in a >10x speedup for partition creation.

Reply:
Beautiful. Maybe just say that then? That is indeed significant. :)

Reply:
I'll check with the tech reviewers about including a number. I saw ">10x faster" in the JIRA ticket scoping the issue, but nothing in the PR or commit message with a specific figure.

Leaving this as `unresolved` for now.